### PR TITLE
Nom nom ugc

### PR DESCRIPTION
**Pull request by joshkay10**
**What's in this PR?**
Additional diluting of logs :(

I understand that this makes our logs less valuable but rules are rules. no UGC/PII

### CODE REVIEW

File: src/jira/client/axios.ts [hunk 1]
```diff
@@ -156,10 +156,10 @@ const instrumentFailedRequest = (baseURL: string, logger: Logger) => {
 				await axios.get("/status", { baseURL });
 			} catch (e) {
 				if (e.response.status === 503) {
-					logger.info(`503 from Jira: Jira instance '${baseURL}' has been deactivated, is suspended or does not exist. Returning 404 to our application.`);
```
Comment (joshkay10) [inline src/jira/client/axios.ts:159]: Move jiraHost out off message and into serialisable attribute

File: src/jira/client/jira-client.ts [hunk 2]
```diff
@@ -206,11 +206,7 @@ export const getJiraClient = async (
 						!withinIssueKeyLimit(data.branches) ||
 						!withinIssueKeyLimit(data.pullRequests)
 					) {
-						logger.warn({
```
Comment (joshkay10) [inline src/jira/client/jira-client.ts:209]: @bgvozdev - This is just a massive list of issue keys, I assumed issue keys fall under UGC but would like a confirm?

File: src/jira/client/jira-client.ts [hunk 3]
```diff
@@ -264,7 +260,6 @@ export const getJiraClient = async (
 						}
 					};
 				}
-				logger?.debug(`Sending builds payload to jira. Payload: ${payload}`);
```
Comment (joshkay10) [inline src/jira/client/jira-client.ts:267]: payload contained much PII

File: src/middleware/github-webhook-middleware.ts [hunk 4]
```diff
@@ -116,8 +116,6 @@ export const GithubWebhookMiddleware = (
 			}
 		});
 
-		context.log.debug({ payload }, "Webhook payload");
```
Comment (joshkay10) [inline src/middleware/github-webhook-middleware.ts:119]: Much PII

**Review by thombergs — APPROVED** [review]

**Comment by atrigueiro** [comment]
:sadpotato:

**Review by atrigueiro — APPROVED** [review]

**Pull request merged**